test/pbn.js

```diff
@@ -367,13 +367,13 @@ describe('PBN', () => {
 
         it('should have multiple lines with braces', (done) => {
             let comment = {};
-            text('{\r\nline 1\r\nline 2\r\n}')
+            text('{\r\nline 1\r\nline 2\r\n\}')
```

Comment:
These two lines are the same.  So why should the test change?

Reply:
Oh my, I don't know how the \ got in there. I'll remove it when I make other changes